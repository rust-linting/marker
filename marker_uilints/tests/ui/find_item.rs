macro_rules! ignore_macro_magic {
    () => {
        static FIND_ITEM: u32 = 4;
    };
}

mod find_real_item {
    static FIND_ITEM: u32 = 4;
}

mod ignore_macro_item {
    ignore_macro_magic!();
}

fn main() {}
