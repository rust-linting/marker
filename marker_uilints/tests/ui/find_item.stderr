warning: hey there is a static item here
 --> $DIR/find_item.rs:8:5
  |
8 |     static FIND_ITEM: u32 = 4;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `duck`
  |
  = note: a note
  = help: a help
note: a spanned note
 --> $DIR/find_item.rs:8:5
  |
8 |     static FIND_ITEM: u32 = 4;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
help: a spanned help
 --> $DIR/find_item.rs:8:5
  |
8 |     static FIND_ITEM: u32 = 4;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  = note: `#[warn(marker::test_lint)]` on by default

warning: 1 warning emitted

